automation/build-artifacts.repos.el7

```diff
@@ -1,2 +1,3 @@
 http://jenkins.ovirt.org/job/ovirt-engine-nodejs_master_build-artifacts-el7-x86_64/lastSuccessfulBuild/artifact/exported-artifacts/
 http://jenkins.ovirt.org/job/ovirt-engine-nodejs-modules_master_build-artifacts-el7-x86_64/lastSuccessfulBuild/artifact/exported-artifacts/
+http://jenkins.ovirt.org/job/ovirt-engine-yarn_master_build-artifacts-el7-x86_64/lastSuccessfulBuild/artifact/exported-artifacts/
```

Comment:
These repos will change when new versions of the packages are built. Currently, for example, the ovirt-engine-nodejs-modules package in this repo is version 1.0.4, not 1.0.3, so the build will fail to install that package. I think that currently the oVirt CI is just silently ignoring this missing packages. I'd suggest you report it. If a package that you request using the `*.req` files isn't available, the build should fail, loudly.